src/ApiService/ApiService/OneFuzzTypes/Events.cs

```diff
@@ -249,25 +248,25 @@ PoolName PoolName
 //        NodeState state
 //        ) : BaseEvent();
 
-//    record EventCrashReported(
-//        Report Report,
-//        Container Container,
-//        [property: JsonPropertyName("filename")] String FileName,
-//        TaskConfig? TaskConfig
-//        ) : BaseEvent();
+record EventCrashReported(
```

Comment:
you will need to update GetventType and GetTypeInfo in BaseEvent to include these